.pylintrc

```diff
@@ -0,0 +1,2 @@
+[Format]
+max-line-length=80
```

Comment:
While we have this long discussion, What would you like to see as disabled? I'm going to disable the "Unnecessary parens after print keyword" warning, as it's a python 3 compatibility thing.